src/app.js

```diff
@@ -436,6 +436,41 @@ var run = function () {
 
   var renderer = new Renderer(editor, executionContext.web3(), updateFiles, udapp, executionContext, formalVerification.event, compiler.event); // eslint-disable-line
 
+  var autoCompile = document.querySelector('#autoCompile').checked;
+
+  document.querySelector('#autoCompile').addEventListener('change', function () {
```

Comment:
This matches the rest of the file. I think it should be updated at a later stage to only use jQuery.